extensions-core/kafka-indexing-service/src/main/java/io/druid/indexing/kafka/KafkaIndexTask.java

```diff
@@ -1056,7 +1057,10 @@ private void sendResetRequestAndWait(Map<TopicPartition, Long> outOfRangePartiti
                                 ));
 
     if (result) {
-      log.warn("Successfully sent the reset request for partitions [%s], waiting to be killed", partitionOffsetMap.keySet());
+      log.makeAlert(
+          "Successfully sent the reset request for partitions [%s], waiting to be killed",
```

Comment:
Many sites use the text of an alert message to group alerts. Since this is an alert message now could you change the text to "Kafka offsets reset for dataSource[%s]" or similar and make the partitions added info through `addData`?